index.bs

```diff
@@ -10588,124 +10592,144 @@ can restart executing this algoritm if needed.
+		4. <a href="#computation-of-value">Compute the value(s)</a> of the
+			{{AudioListener}}'s {{AudioParam}}s for this block.
 
-		7. Add <em>node</em> to the beginning of <em>ordered</em>.
+		5. For each {{AudioNode}}, in the order determined previously:
 
-	5. Reverse the order of <em>nodes</em>.
+			1. For each {{AudioParam}} of this {{AudioNode}}, execute these steps:
 
-	6. For each {{DelayNode}} in a cycle, <a href="#available">make
-		available for reading</a> a block of audio from the {{DelayNode}}
-		buffer, available for reading.
+				1. If this {{AudioParam}} has any {{AudioNode}} connected to it,
+					<a href="#SummingJunction">sum</a> the buffers
+					<a href="#available-for-reading"> made available for reading</a> by
+					all {{AudioNode}} connected to this {{AudioParam}},
+					<a href="#down-mix">down mix</a> the resulting buffer down to a mono
+					channel, and call this buffer the <dfn id="input-audioparam-buffer">
+					input AudioParam buffer</dfn>.
 
-	7. <a href="#computation-of-value">Compute the value(s)</a> of the
-		{{AudioListener}}'s {{AudioParam}}s for this block.
+				2. <a href="#computation-of-value">Compute the value(s)</a> of this
+					{{AudioParam}} for this block.
 
-	8. For each {{AudioNode}}, in the order determined previously:
-		1. For each {{AudioParam}} of this {{AudioNode}}, execute
-			these steps:
+				3. <a>Queue a control message</a> to set the {{[[current value]]}} slot
+					of this {{AudioParam}} to the last value computed in the preceding
+					step.
 
-			1. If this {{AudioParam}} has any {{AudioNode}}
-				connected to it, <a href="#SummingJunction">sum</a> the
-				buffers <a href="#available">made available for reading</a>
-				by all {{AudioNode}} connected to this {{AudioParam}},
-				<a href="#down-mix">down mix</a> the resulting buffer down to
-				Mono, and call this buffer the <dfn id="input-audioparam-buffer">input AudioParam
-				buffer</dfn>.
+			2. If this {{AudioNode}} has any {{AudioNode}}s connected to its input,
+				<a href="#SummingJunction">sum</a> the buffers
+				<a href="#available-for-reading">made available for reading</a> by all
+				{{AudioNode}}s connected to this {{AudioNode}}. The resulting buffer is
+				called the <dfn>input buffer</dfn>.
+				<a href="#channel-up-mixing-and-down-mixing">Up or down-mix</a> it to
+				match if number of input channels of this {{AudioNode}}.
 
-			2. <a href="#computation-of-value">Compute the value(s)</a> of
-				this {{AudioParam}} for this block.
+			3. If this {{AudioNode}} is a <a>source node</a>,
+				[=Computing a block of audio|compute a block of audio=], and
+				<a href="#available-for-reading">make it available for reading</a>.
 
-			3. <a>Queue a control message</a> to set the
-				{{[[current value]]}} slot of this {{AudioParam}}
-				to the last value computed in the preceding step.
+			4. If this {{AudioNode}} is an {{AudioWorkletNode}}, invoke associated
+				{{AudioWorkletProcessor}}'s {{process}} method to
```

Comment:
You need to fix this bikeshed error about `{{process}}` not existing, which is true. It's not defined by IDL.

Reply:
I did what I could. PTAL at l.9812 in the latest patch.